cmeutils/structure.py

```diff
@@ -63,11 +81,16 @@ def angle_distribution(
                 v = pos3_unwrap - pos2_unwrap
                 angles.append(np.round(angle_between_vectors(u, v, False), 3))
     trajectory.close()
-    return np.array(angles)
+
+    if histogram:
+        bin_centers, bin_heights = get_histogram(np.array(angles), bins=bins)
+        return np.stack((bin_centers, bin_heights)).T
+    else:
+        return np.array(angles)
 
 
 def bond_distribution(
-    gsd_file, A_name, B_name, start=0, stop=-1
+    gsd_file, A_name, B_name, start=0, stop=-1, histogram=False, bins=100
```

Comment:
We might want to expose the normalize kwarg here if we want to use this as a function wrapper or just use the two functions separately.